File: week1/pset1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int height, int bricks);

int main(void)
{

    int n;
    // prompt user for positive integer
    do
    {
        n = get_int("Size: ");
    }
    while (n < 1 || n > 8);

    // Print an n-by-n grid of bricks
    for (int i = 0; i < n; i++)
    {
        print_row(n, i + 1);
    }
}

void print_row(int height, int bricks)
{
    for (int i = 0; i < height - bricks; i++)
    {
        printf(" ");
    }
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: week6/pset6/sentimental-mario-less/mario.py
from cs50 import get_int


def print_row(height, bricks):
    for i in range(height - bricks):
        print(" ", end="")
    for i in range(bricks):
        print("#", end="")
    print()


def main():
    n = get_int("Digite uma altura:")
    while n < 1 or n > 8:
        n = get_int("Digite uma altura:")

    for i in range(n):
        print_row(n, i + 1)


if __name__ == "__main__":
    main()
